cardano-db/src/Cardano/Db/Query.hs

```diff
@@ -234,7 +235,7 @@ queryIsFullySynced = do
       let psec = diffUTCTime currentTime latestBlockTime
       -- SLot durection is in microseconds, and the difference from current time to
```

Comment:
```suggestion
      -- Slot duration is in microseconds, and the difference from current time to
```

Reply:
It still says "durection" now, is that some special term I don't know? :slightly_smiling_face:

Reply:
I really need a comment spell checker.